src/main/java/ua/utility/kfsdbupgrade/md/base/TimedInterval.java

```diff
@@ -0,0 +1,126 @@
+package ua.utility.kfsdbupgrade.md.base;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.System.currentTimeMillis;
+import static java.util.Objects.hash;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+import com.google.common.base.Stopwatch;
+import com.google.common.collect.ComparisonChain;
+
+/**
+ * Holds information related to a period of time.
+ *
+ * Stop is guaranteed to be greater than or equal to start.
+ *
+ * Elapsed is guaranteed to be the difference between stop and start (might be zero)
+ * 
+ * @author Jeff Caddel
+ */
+public final class TimedInterval implements Comparable<TimedInterval> {
+

```

Comment:
It appears this class isn't being used; do you have near term plans to do so?

Reply:
Was used at various points along the way to help capture and display timing information, if it's not actually being used at the moment, it might be soon.